### PR TITLE
hi.json file modified to latest, till beginShape()

### PR DESCRIPTION
**Pull request by SanjaySinghRajpoot**
hi.json file earlier only had paragraphs without any hyperlinks now I have added hyperlinks to the text with comments and translated.

An outcome of Google Summer of Code 2021
Author- Sanjay Singh Rajpoot @SanjaySinghRajpoot
Mentor - Aditya Rana @ranaaditya

### CODE REVIEW

File: src/data/reference/hi.json [hunk 1]
```diff
@@ -112,7 +112,11 @@
 			"returns":"p5 ऑब्जेक्ट"
 		},
 		"color":{
-			"description":"प्रकार के रंग के चर में संग्रहीत किए जाने वाले रंग बनाएं। मापदंडों को RGB या HSB मानों के रूप में व्याख्या किया जाता है, जो colorMode के अनुसार वर्तमान रंग मोड पर निर्भर करता है) ()। डिफ़ॉल्ट मोड 0 और 255 के बीच मानों के साथ RGB है, इसलिए, रंग फ़ंक्शन (255, 204, 0) एक चमकीले पीले रंग देता है। ध्यान दें कि यदि रंग () फ़ंक्शन को केवल एक मूल्य प्रदान किया जाता है, तो इसे ग्रेस्केल मूल्य के रूप में व्याख्या किया जाएगा। एक दूसरा मान जोड़ें, और इसका उपयोग अल्फा पारदर्शिता के रूप में किया जाएगा। जब तीन मान निर्दिष्ट किए जाते हैं, तो उन्हें RGB या HSB मान के रूप में व्याख्या की जाती है। चौथा मान जोड़ते समय, अल्फा पारदर्शिता लागू की जाती है। यदि केवल एक स्ट्रिंग पैरामीटर प्रदान किया जाता है, तो इसे सीएसएस संगत रंग स्ट्रिंग के रूप में व्याख्या किया जाएगा। रंगों को संख्या या व्यवस्था के रूप में संग्रहीत किया जाता है।",
+			"description":[
```
Comment (ranaaditya) [inline src/data/reference/hi.json:115]: Don't translate function names into hindi !
Revert the changes needed !

File: src/data/reference/hi.json [hunk 2]
```diff
@@ -196,7 +218,11 @@
             "returns": "p5 ऑब्जेक्ट"
         },
         "fill": {
-            "description": "ज्यामितीय आंकड़े भरने के लिए उपयोग किए जाने वाले रंग को परिभाषित करता है। उदाहरण के लिए, यदि आप भरण (204, 102, 0) चलाते हैं, तो नीचे दिए गए सभी आंकड़ों में नारंगी भरण होगा। यह रंग RGB या HSB रंग के संदर्भ में निर्दिष्ट है, colorMode () के अनुसार रंग मोड पर निर्भर करता है (डिफ़ॉल्ट रंग डोमेन RGB है, प्रत्येक मान 0 और 255 के बीच की सीमा में है)। यदि कोई स्ट्रिंग प्रकार तर्क दिया जाता है, तो RGB, RGBA और हेक्साडेसिमल CSS प्रकार समर्थित हैं। भराव के रंग को परिभाषित करने के लिए एक कलर पी 5 ऑब्जेक्ट प्रदान किया जा सकता है।",
+            "description": [
+                "आकृतियों को भरने के लिए प्रयुक्त रंग सेट करता है। उदाहरण के लिए, यदि आप भरण (204, 102, 0) चलाते हैं, तो भरण आदेश के बाद खींची गई सभी आकृतियों को नारंगी रंग से भर दिया जाएगा। यह रंग या तो आरजीबी के संदर्भ में निर्दिष्ट है या वर्तमान <a href=\"#/p5/colorMode\">colorMode()</a> के आधार पर HSB रंग। (डिफ़ॉल्ट रंग स्थान RGB है, जिसमें प्रत्येक मान 0 से 255 के बीच होता है)। अल्फा डिफ़ॉल्ट रूप से सीमा भी 0 से 255 तक होती है।",
```
Comment (ranaaditya) [inline src/data/reference/hi.json:222]: Don't translate the function names into hindi please ;

File: src/data/reference/hi.json [hunk 3]
```diff
@@ -426,7 +500,7 @@
             "returns": "p5 ऑब्जेक्ट"
         },
         "curveTangent": {
-            "description": "बिंदु a, b, c, d के लिए स्थिति t पर वक्र की स्पर्शरेखा का मूल्यांकन करें। पैरामीटर टी 0 और 1 के बीच भिन्न होता है, ए और डी वक्र के बिंदु हैं, बी और सी नियंत्रण बिंदु हैं।",
+            "description": "ए, बी, सी, डी के लिए स्थिति टी पर वक्र के स्पर्शरेखा का मूल्यांकन करता है। पैरामीटर टी 0 और 1 के बीच भिन्न होता है, ए और डी वक्र पर बिंदु होते हैं, और बी और सी नियंत्रण बिंदु होते हैं।",
```
Comment (ranaaditya) [inline src/data/reference/hi.json]: Don't translate the function names, variable names, constant names , etc into hi di please !

**Review by ranaaditya — CHANGES_REQUESTED** [review]

File: src/data/reference/hi.json (hunk 1 above)
Comment (SanjaySinghRajpoot) [reply src/data/reference/hi.json:115]: ok

File: src/data/reference/hi.json (hunk 2 above)
Comment (SanjaySinghRajpoot) [reply src/data/reference/hi.json:222]: ok

File: src/data/reference/hi.json (hunk 3 above)
Comment (SanjaySinghRajpoot) [reply src/data/reference/hi.json]: ok
